deltachat-ios/Chat/ChatViewController.swift

```diff
@@ -1673,14 +1709,28 @@ class ChatViewController: UITableViewController {
     }
 
     // MARK: - Context menu
-    private func prepareContextMenu() {
-        UIMenuController.shared.menuItems = contextMenu.menuItems
+    private func prepareContextMenu(for message: DcMsg, isHidden: Bool) {
+        if #available(iOS 13.0, *) {
+            return
+        }
+
+        if isHidden {
+            UIMenuController.shared.menuItems = nil
+        } else if isGroupChat && !message.isFromCurrentSender {
+            UIMenuController.shared.menuItems = contextMenu.menuItems
+        } else {
+            // filter out replyPrivatelyItem
```

Comment:
this comment seems superfluous as it just says repeats what is visible in the code